crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1770,21 +1770,16 @@ impl World {
         f: impl FnOnce(&mut World, &mut Schedule) -> R,
     ) -> Result<R, TryRunScheduleError> {
         let label = label.as_ref();
-        let Some((extracted_label, mut schedule)) = self
+        let Some((_, mut schedule)) = self
             .get_resource_mut::<Schedules>()
             .and_then(|mut s| s.remove_entry(label))
```

Comment:
Should `remove_entry` still return the label?

Reply:
I'm not sure if `remove_entry` has any use. We already have `remove`, so we should just delete `remove_entry` if it doesn't. It's slightly useful in that you can get the owned Boxed label for free, but not sure that's worth the added api. 

I should switch to using `remove` here, since we're not using the label.

Reply:
Fair enough. If #7762 gets merged then we should definitely get rid of `remove_entry`, since there'd be no benefit to keeping the schedule label.